src/prisma/schema.prisma

```diff
@@ -2570,3 +2570,10 @@ model GitcoinProjectAttestation {
   schemaId       String
   attestationUID String
 }
+
+model ServiceWorkerSubscriptions {
+  id           String    @id @default(uuid()) @db.Uuid
+  createdAt    DateTime  @default(now())
+  deletedAt    DateTime?
```

Comment:
is there any reason to not just delete these?

Reply:
Nope. I will remove it.